app/views/main.scala.html

```diff
@@ -13,7 +16,8 @@
 	styles: Html = Html(""),
 	scripts: Html = Html(""),
   data: Map[String, String] = Map.empty,
-  csrf: Option[String] = None
+  csrf: Option[String] = None,
+  productPrices: Option[ProductPrices] = None,
```

Comment:
I'm not sure this needs to be an `Option` it's not on the checkout page

Reply:
`main` is used for all default responses, most without prices right now and some that will never have prices (like 404 pages or the checkout ty) I'd rather not create a new html template if we can avoid it

Reply:
I hadn't noticed that this was in main. It would probably be good to remove the code from `digitalSubscription.scala.html` which does the same thing and delegate to this file.

Reply:
good shout!